index.js

```diff
@@ -35,23 +37,19 @@ module.exports = class{
   }
 
   getFieldFormValue(attrName) {
-    if(isArray(this.parsedData[attrName])) {
-      return(this.parsedData[attrName]);
-    } else if(isEmpty(this.formData[attrName]) && !isNumber(this.formData[attrName])) {
+    let data = get(this.formData, attrName);
```

Comment:
This "get" and "set" are the backbones to deep nested data in the FormLinker. Gives the ability to set and fetch data nested deep without having to iterate over the data sets a whole bunch.